strata.default.lua

```diff
@@ -110,8 +83,31 @@ strata.set_config {
 	animations = {
 		enabled = true,
 	},
-
-	-- I guess we can have the same systems as above here for convenience for most users:
-	bindings = { ... }, -- same thing as the argument for `strata.set_bindings()`
-	rules = { ... }, -- same thing as the argument for `strata.set_rules()`
+	bindings = {
+		{
+			keys = { "CTRL", "SHIFT", "Q" },
+			cmd = close_all_windows,
+		},
+		{
+			keys = { "WIN", "RETURN" },
+			cmd = strata.cmd.spawn("kitty --title Terminal"),
+		},
+		{
+			keys = { "WIN", "SPACE" },
+			cmd = strata.cmd.spawn("rofi --show drun")
+		},
+  },
```

Comment:
There's some weird indentation glitch going on. And I see you're using spaces while I was using tabs :sweat_smile: 

We should agree on a way to format the code (I was using `stylua`, and I already committed a `.stylua.toml` file, which we can remove if you prefer), and while we're at it also add a `.editorconfig` file so that our different editors agree. Same for rustfmt, probably add some config file with your preferred settings to make sure we're all on the same page.

Reply:
You're right. I already have a `.rustfmt.toml` file. I was out of the NeoVim league for a while and I was just setting up AstroNvim so thats probably why the indentation is off. I do prefer tabs over spaces. Also I've never heard of `.editorconfig`. I'll look it up and add it.

Reply:
Awesome, thanks! If you're using neovim, I recommend [this plugin](https://github.com/elentok/format-on-save.nvim) to handle your auto-formatting needs. I'm using the LSP formatting for rust, and stylua for lua (both are built-in in the plugin).

`.editorconfig` is an editor-agnostic file that lets you declare some very basic settings, such as "tabs or space", ends of lines, etc. Here's what I usually have in all my projects:

```editorconfig
root = true

[*]
indent_style = tab
indent_size = 4
end_of_line = lf
trim_trailing_whitespace = true
insert_final_newline = true
```

Reply:
I added one a few minutes back but I'll add these changes and update it. Cheers :)